### PR TITLE
ISSUE-864 Remove esv6 support

### PR DESCRIPTION
**Pull request by Arsnael**
Last commit depends on @vttranlina  fixing the build in https://github.com/linagora/tmail-backend/pull/874

### CODE REVIEW

**Review by chibenwa — APPROVED** [review]
Didnt read the diff, but love it.

**Review by vttranlina — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -53,7 +53,7 @@ pipeline {
 
               echo "Docker tag: ${env.DOCKER_TAG}"
               // build and push docker images
-              sh "mvn -Pci jib:build -pl apps/distributed,apps/distributed-es6-backport,apps/memory -X"
+              sh "mvn -Pci jib:build -pl apps/distributed,apps/memory -X"
```
Comment (vttranlina) [inline Jenkinsfile:56]: you can forget it, it will prevent conflict with https://github.com/linagora/tmail-backend/pull/874

Comment (Arsnael) [reply Jenkinsfile]: Yep I dropped the wip commit :)

**Comment by vttranlina** [comment]
```
java.util.concurrent.ExecutionException: org.testcontainers.containers.ContainerLaunchException: Container startup failed for image opensearchproject/opensearch:2.1.0
```
It looks unstable ci, I will restart it

**Pull request merged**